app/Http/Controllers/NotesController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Actionlog;
+use App\Models\Asset;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Validation\Rule;
+
+class NotesController extends Controller
+{
+    public function store(Request $request)
+    {
+        $this->authorize('update', Asset::class);
+
+        $validated = $request->validate([
+            'id' => 'required',
+            'note' => 'required|string|max:500',
```

Comment:
This 500 limit is carry over from the previous implementation. That's not that much...(the text from my PR description is over 500). Should we bump that up?

Right now validation errors redirect back with the "there was problems with the form below" message but the modal doesn't pop again...

@snipe something to consider before merging.

Reply:
Hrm. Well there’s not going to be many times that you’d run into a validation error on notes, since the validation is pretty loose.